packages/backend-core/src/users/users.ts

```diff
@@ -39,6 +40,27 @@ function removeUserPassword(users: User | User[]) {
   return users
 }
 
+export const isSupportedUserSearch = (query: SearchQuery) => {
+  const allowed = [
+    { op: SearchQueryOperators.STRING, key: "email" },
+    { op: SearchQueryOperators.EQUAL, key: "_id" },
+  ]
+  for (let [key, operation] of Object.entries(query)) {
+    if (typeof operation !== "object") {
+      return false
+    }
+    const fields = Object.keys(operation || {})
+    const allowedOperation = allowed.find(
+      allow =>
+        allow.op === key && fields.length === 1 && fields[0] === allow.key
+    )
+    if (!allowedOperation && fields.length > 0) {
```

Comment:
This is a little bit confusing. Will we ever have more than one field? On the line above we are checking that we only have one

Reply:
The idea is to support something like `range: {}` where the operator has been supplied but with no search parameters. This is allowable as it technically doesn't nothing, but I could probably do something like stripping them out or skipping them which would make more sense.